packages/seacas/libraries/ioss/src/Ioss_DynamicTopology.C

```diff
@@ -415,6 +842,47 @@ std::string DynamicTopologyFileControl::get_unique_filename(Ioss::DatabaseUsage
   return filename;
 }
 
+std::string DynamicTopologyFileControl::get_change_set_name(unsigned int step)
+{
+  std::ostringstream change_setname;
+  change_setname << change_set_prefix();
+  change_setname << step;
+  return change_setname.str();
+}
+
+std::string DynamicTopologyFileControl::get_cyclic_database_filename(const std::string& baseFileName,
+                                                                           unsigned int fileCyclicCount,
+                                                                           unsigned int step)
+{
+  std::string filename = baseFileName;
+
+  if(fileCyclicCount > 0)
+  {
+    // The file suffix cycles through the first fileCyclicCount'th entries in A,B,C,D,E,F,...
+    if(step == 0)
+      step++;
+
+    static std::string suffix = "ABCDEFGHIJKLMNOPQRSTUVWXYZ";
```

Comment:
Seems like we have this in multiple places...  Maybe one is for input and one for output...

Reply:
Are you talking about line 865 or the function?

Reply:
The line is what caught my attention, but I think the same functionality of generating a cyclic file name is present in many locations; especially now that we have pulled some framework code down into IOSS.  Maybe now is time to consolidate into a single function if possible.

Reply:
Ok, I'll look into consolidation